before.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# The required environment variables are:
+# user - e.g. 'pjmhill'
+
+# optional variables for github commit status API
+# github_token - the token that is used to push commit status
```

Comment:
can you add guidance as to how to get a github token (perhaps link to the dev' bootstrap)?